ray-operator/controllers/ray/common/pod_test.go

```diff
@@ -843,3 +819,93 @@ func TestDefaultInitContainer(t *testing.T) {
 	assert.NotEmpty(t, rayContainer.Resources, "The test only makes sense if the Ray container has resource limit/request.")
 	assert.Equal(t, rayContainer.Resources, healthCheckContainer.Resources)
 }
+
+func TestSetMissingRayStartParamsAddress(t *testing.T) {
+	// The address option is automatically injected into RayStartParams with a default value of <Fully Qualified Domain Name (FQDN)>:<headPort> for only workers (used for connecting to the Ray cluster).
+	// Head should not include address option in RayStartParams as it can be accessed via localhost.
```

Comment:
Head should not include address option in RayStartParams as it can be accessed via localhost because GCS server is running on the head Pod.